contrib/build-push-ceph-container-imgs.sh

```diff
@@ -248,13 +250,13 @@ function push_ceph_imgs_latest {
     local_tag=${CONTAINER_REPO_ORGANIZATION}/daemon-base:${RELEASE}-${BRANCH}-centos-${CENTOS_RELEASE}-${HOST_ARCH}
     full_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${RELEASE}-centos-${CENTOS_RELEASE}-${HOST_ARCH}-devel
     branch_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${BRANCH}
-    sha1_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${SHA1}
+    sha1_flavor_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${SHA1}-${OSD_FLAVOR}
     docker tag $local_tag $full_repo_tag
     docker tag $local_tag $branch_repo_tag
-    docker tag $local_tag $sha1_repo_tag
+    docker tag $local_tag $sha1_flavor_repo_tag
     docker push $full_repo_tag
```

Comment:
@dmick can you help verify the use-case for full_repo_tag + branch_repo_tag or are they obsolete now?

Reply:
They were always for convenience so a developer can configure his branch or a specific sha1.  I don't see anything that would make them obsolete.

Reply:
@dmick can we add a flavor postfix to their tag names?  just to differentiate the default flavor build from the crimson flavored build? our concern is that it might break some users of these images, if we change the naming scheme of them.

Reply:
It would probably be safer to add a few new tags that include crimson.  I don't know for sure who is using what tags for what reason, but adding the capability to select crimson specifically seems safest.